i18n/en.pot

```diff
@@ -118,6 +121,12 @@ msgstr ""
 msgid "Number of Periods"
 msgstr ""
 
+msgid "{{helpText}}"
+msgstr ""
+
+msgid "{{label}}"
+msgstr ""
+
```

Comment:
@Sharmyn28 these new strings are not helpful at all for translators, since they only include a variable.  They should probably be updated to actually show the translation or removed